catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -125,11 +138,16 @@
                             platform-util,
                             platform-util-unavailableurls,
                             failsafe,
-                            hazelcast
+                            hazelcast,
+                            sardine,
+                            httpclient,
```

Comment:
It looks like we do this in a lot of places. I tried without the embeds initially and their packages were not available. It looks like technical debt to make them available in the container and remove all the embeds.